src/routes/(split)/components/tooltip/+page.md

```diff
@@ -1,19 +1,25 @@
 <script lang="ts">
   import Tooltip from "$lib/components/Tooltip.svelte";
   import Checkbox from "$lib/components/Checkbox.svelte";
+  import Input from "$lib/components/Input.svelte";
 
   let overflowHidden = true;
   let showShowcase = true;
+  let tooltipText = "This button is disabled because of a long and complicated explanation.";
```

Comment:
Why did you change the text?

Reply:
That was an accident that I only realized when updating the screenshot.